docs/topics/auth/default.txt

```diff
@@ -1654,7 +1654,7 @@ provides several built-in forms located in :mod:`django.contrib.auth.forms`:
     A form used in the admin interface to change a user's information and
     permissions.
 
-.. class:: UserCreationForm
+.. class:: BaseUserCreationForm
```

Comment:
Needs a `versionadded` annotation, and should say ≈"This is the recommended base class if you need to customise the user creation form."